grails-app/services/org/pih/warehouse/core/MailService.groovy

```diff
@@ -201,7 +202,7 @@ class MailService {
             email.send()
             return true
         } catch (Exception e) {
-            log.error("could not send ${summary}", e)
+            log.error("could not send ${summary}: ${ExceptionUtil.summarize(e)}")
```

Comment:
Instead of dumping hundreds of lines of javax.mail traceback, we now just pull out the "Caused by" lines. Much easier for a human to read

Reply:
I'm not sure I agree with optimizations like this. As an example, I just ran into an issue where java decided to disappear stacktraces of NPEs after the first occurrence in the log file. Since I didn't have access to the server or logs, I had no way of finding out what the stacktrace was so I had to do a ton of work just to figure it out. In general, I prefer verbosity over The Nothing so unless this was 100% guaranteed to remove obfuscation without potentially causing important bits to disappear, then we shouldn't do it.

https://pihemr.atlassian.net/browse/OBKN-668?focusedCommentId=133280

Reply:
Resolved in e7740e183